Library/Formula/bork.rb

```diff
@@ -0,0 +1,14 @@
+class Bork < Formula
+  desc "Bash DSL for config management"
+  homepage "https://github.com/mattly/bork"
+  url "https://github.com/mattly/bork/archive/v0.9.tar.gz"
+  sha256 "3e8217d5fe4de94663dac74af505d05649c9d995998d19d4b247e5d3ecb1dd4b"
+
+  def install
+    prefix.install %w[bin docs lib test types]
+  end
+
+  test do
+    assert_match /bork usage:/, shell_output("#{bin}/bork", 1)
```

Comment:
We might want to use a directory in the test path; otherwise it’ll try to create `/usr/local/bin` if it doesn’t exist. Something like that should work:
```rb
test do
  system "#{bin}/bork", "do", "ok", "directory", "#{testpath}/foo"
  assert (testpath/"foo").directory?
end
```